src/lib/Behat/PageElement/Fields/RichText.php

```diff
@@ -57,4 +58,103 @@ public function verifyValueInItemView(array $values): void
             'Field has wrong value'
         );
     }
+
+    public function openElementsToolbar()
+    {
+        $this->context->findElement($this->fields['addButton'])->click();
+    }
+
+    public function changeStyle(string $style)
+    {
+        if (!in_array($style, self::ALLOWED_STYLE_OPTIONS)) {
+            throw new Exception(sprintf('Unsupported style: %s', $style));
+        }
+
+        $this->context->findElement($this->fields['styleDropdown'])->click();
+        $this->context->findElement(sprintf('.ae-listbox li %s', $style))->click();
+    }
+
+    public function insertNewLine(): void
+    {
+        $this->getFieldInput();
+        $this->context->getSession()->getDriver()->executeScript(sprintf($this->executeAlloyEditorScript, $this->richtextId, 'enter'));
+    }
+
+    public function insertLine($value, $style = ''): void
+    {
+        $this->getFieldInput();
+        $this->context->getSession()->getDriver()->executeScript(sprintf($this->insertAlloyEditorValueScript, $this->richtextId, $value));
+
+        if ($style !== '') {
+            $this->changeStyle($style);
+            Assert::assertContains(sprintf('%s%s</%s>', $value, '<br>', $style), $this->context->findElement(sprintf('.ez-data-source__richtext %s', $style))->getOuterHtml());
+        }
+    }
+
+    private function getFieldInput(): NodeElement
+    {
+        $fieldInput = $this->context->findElement($this->fields['fieldInput']);
+        Assert::assertNotNull($fieldInput, sprintf('Input for field %s not found.', $this->label));
+        $this->richtextId = $fieldInput->getAttribute('id');
+
+        return $fieldInput;
+    }
+
+    public function clickOnRichTextEditor()
```

Comment:
Is it used? I can't find any usages in this PR or in https://github.com/ezsystems/ezplatform-page-builder/pull/421/files (but I might be missing something)